js/views/modals/About/About.js

```diff
@@ -0,0 +1,100 @@
+import $ from 'jquery';
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import Story from './Story';
+import Contributors from './Contributors';
+import Donations from './Donations';
+import License from './License';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      removeOnRoute: false,
```

Comment:
Okay.

I guess the special reason it exists in Settings is so that modal stays open when something like #connected-peers is routed to.